File: library/src/main/java/net/robinx/queue/JobHandlerListenerAdapter.java
package net.robinx.queue;

/**
 * Created by Robin on 2016/6/28 18:12.
 */
public class JobHandlerListenerAdapter<T extends Job<T>> implements JobHandlerListener<T> {
    @Override
    public void onPrepare(T job) {

    }

    @Override
    public void onCancel(T job) {

    }

    @Override
    public void onFinish(T job) {

    }
}

File: library/src/main/java/com/github/jobqueue/JobHandlerListenerAdapter.kt
package com.github.jobqueue

/**
 * Created by Robin on 2016/6/28 18:12.
 */
open class JobHandlerListenerAdapter<T : Job<T>> : JobHandlerListener<T> {
    override fun onPrepare(job: T) {
    }

    override fun onCancel(job: T) {
    }

    override fun onFinish(job: T) {
    }
}
